index.html

```diff
@@ -4,15 +4,86 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    
     <!-- Add a link to your custom CSS here -->
 
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+<main class="h-main">
 
+     <section class="h-hero">
+       <h2>
+        <input type="checkbox">Maya Nelson</h2>
+       <p>PROJECT MANAGER</p>
+     </section>
+
+     <section class="h-feature">
+        
+          <img src="maya.jpeg" alt="maya">
+          <h2>Maya</h2>
+          <h2 >Nelson</h2>
+          <button class="one"><a id="left" href="#">RESUME</a></button> <button class="two" ><a id="right" href="#">PROJECTS</a></button><br>
+        
+        <div class="logo">
+          <img src="facebook-icon.svg" alt="facebook" width="20">
```

Comment:
excellent alt text here. Would these be links on a real page?

Reply:
These 4 logos need to link to real pages.